tests/python/contrib/test_ethosn/test_conv2d.py

```diff
@@ -228,10 +227,7 @@ def test_conv2d_depthwise(
             )
         ),
     }
-    if ethosn_api_version() == "3.2.0":
-        input_zp = np.random.randint(0, np.iinfo(dtype).max)
-    else:
-        input_zp = np.random.randint(np.iinfo(dtype).min, np.iinfo(dtype).max)
+    input_zp = np.random.randint(0, np.iinfo(dtype).max)
```

Comment:
Out of interest, was there a reason for requiring this and the below change?

Reply:
Driver 22.11 has a bug related to zero points that is being tracked by the NPU driver.